web/pdf_page_view.js

```diff
@@ -468,19 +472,15 @@ class PDFPageView {
     if (!this.textLayer) {
       return;
     }
-    this.structTreeLayer ||= new StructTreeLayerBuilder();
 
-    const tree = await (!this.structTreeLayer.renderingDone
-      ? this.pdfPage.getStructTree()
-      : null);
-    const treeDom = this.structTreeLayer?.render(tree);
-    if (treeDom) {
+    const treeDom = await this.structTreeLayer.render();
```

Comment:
Please let's ensure that that the structTreeLayer is actually available when the method is invoked.
```suggestion
    const treeDom = await this.structTreeLayer?.render();
```